x-pack/legacy/plugins/monitoring/server/routes/api/v1/logstash/pipelines/cluster_pipelines.js

```diff
@@ -47,15 +78,25 @@ export function logstashClusterPipelinesRoute(server) {
       ];
 
       try {
+        const pipelineData = await getPipelines(req, lsIndexPattern, pipelineIds, metricSet);
+        // We need to re-sort because the data from above comes back in random order
```

Comment:
One suggestion to avoid re-sorting: We're passing in the `pipelineIds` array, which already has the pipeline IDs in the sorted order, to `getPipelines` now. Inside `getPipelines` (specifically right around [this part of the code](https://github.com/elastic/kibana/blob/master/x-pack/legacy/plugins/monitoring/server/lib/logstash/get_pipelines.js#L43)), you can use this array to return the pipelines in the right order.

Reply:
Good idea. Done